docs/1.0. Certificate Provisioning.md

```diff
@@ -67,47 +67,48 @@ The entity that is using the NMOS API, for example:
 
 ## Introduction (informative)
 
-This document covers the automated provisioning of TLS Certificates to NMOS Servers, which are then used to secure communications between NMOS Servers and NMOS Clients.
+This document covers the automated provisioning of TLS Certificates to NMOS Servers and automated provisioning of CA Certificates to NMOS Clients. These certificates are used to secure communication between NMOS Servers and NMOS Clients.
 
 This document is not concerned with the security of the connection used to carry out provisioning of the TLS Certificate, but for the mechanism described in this document to be effective the connection must be secured, ideally using the recommendations covered in [BCP-003-01][].
 
```

Comment:
```suggestion

## NMOS Client Automated Certificate Provisioning Flow (informative)

To allow NMOS Client to trust the NMOS Server. NMOS Client must initially obtain the Root Certificate Authority certificates, as the chain of trust, which signed the NMOS Server certificate.  NMOS Client begins by following the chain tracing backward until it arrives at a trusted root certificate. If the certificate is valid and can be chained back to a trusted root, NMOS Server will be trusted. If it can’t be chained back to a trusted root, the NMOS Client will reject the communication with the NMOS Server.

1. When the NMOS Client (EST Client) is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
2. The NMOS Client will then request the Root CA for the target network, from the EST Server.
   - The NMOS Client can assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
   - The NMOS Client uses the Root CA returned to secure further communications with the EST Server and the NMOS Servers.

The workflow if the NMOS Client is moved to a new network is covered in detail in the following [sections](#nmos-client).

![NMOS Client Workflow](images/nmos_client_workflow.png)
```